sources/api/bootstrap-containers/src/main.rs

```diff
@@ -92,18 +92,21 @@ const ENV_FILE_DIR: &str = "/etc/bootstrap-containers";
 const DROPIN_FILE_DIR: &str = "/etc/systemd/system";
 const PERSISTENT_STORAGE_DIR: &str = "/local/bootstrap-containers";
 const DROP_IN_FILENAME: &str = "overrides.conf";
+const DEFAULT_CONFIG_PATH: &str = "/etc/bootstrap-containers/bootstrap-containers.toml";
 
 /// Stores user-supplied global arguments
 #[derive(Debug)]
 struct Args {
     log_level: LevelFilter,
+    config_path: PathBuf,
     socket_path: String,
 }
 
 impl Default for Args {
     fn default() -> Self {
         Self {
             log_level: LevelFilter::Info,
+            config_path: PathBuf::from_str(DEFAULT_CONFIG_PATH).unwrap(),
```

Comment:
We need to clean up the other use of `socket_path` in `mark_bootstrap` - the larger goal of this series of changes is to remove the datastore dependency from these programs entirely.

It's going to be a bit more involved. What I'd suggest trying is calling `apiclient set bootstrap-containers.%i=off` instead. Not sure that will work but maybe.

Reply:
Doesn't this still not solve the problem? Since the apiclient call and apiclient set commands would essentially be identical operations and dependency only difference is calling the apiclient binary instead of direct sending the request?